decentraland/unity-renderer
# style: fix amount label horizontal position

This PR was created to fix the backpack item's amount label position. It must always be aligned at the bottom right corner, independently the width of the item. This go broken after updating the Backpack width to fit all resolutions.

<img width="472" alt="Screenshot 2023-11-14 at 15 18 09" src="https://github.com/decentraland/unity-renderer/assets/51088292/6df475e2-4bcf-4a72-824d-c25916ea67f5">

### How to test
`To test this PR you must have more than one item of the same asset`

1- Open this [link](https://play.decentraland.org/?explorer-branch=style/ItemAmountLabelHorizontalPositionDynamic)
2- Open the backpack pressing `I` or vía the menu button.
3- Check the label position is properly placed in the item where you have multiple assets.
